pkg/util/util.go

```diff
@@ -48,7 +52,7 @@ func IsVersionGreaterThan(version1, version2 string) (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	return semanticVersion2.LessThanOrEqual(semanticVersion1), nil
+	return semanticVersion1.GreaterThan(semanticVersion2), nil
```

Comment:
Lets  add unit test with the bug symptoms  and before the change that test should fail and after change it should pass.

Reply:
ok